### ISSUE TITLE
Footnotes should not be displayed in Post excerpts

### ISSUE THREAD

**Issue by Rumperuu** [markcheret/footnotes#47]
**Describe the bug**
When Post excerpts are displayed, the footnotes icons are still being rendered.

**To Reproduce**
1. Load a Theme that displays Post excerpts
2. Create a test post with footnotes in the first few lines
3. View the excerpt

**Expected behavior**
Footnote tags should be hidden in Post excerpts.

**Screenshots**
[https://imgur.com/a/eIo9EYI](url)

**Desktop (please complete the following information):**
 - Plugin: 2.5.6

**Comment by lolzim** [comment]
Makes sense. This illustrate the possible need to distribute tasks among us, IMHO.

**Comment by Rumperuu** [comment]
The reference container is also being rendered in post excerpts (although this is less of a concern as it's unlikely that anyone will have a post short enough to show it.

![Screenshot from 2021-04-13 08-13-13](https://user-images.githubusercontent.com/5621904/114794728-ef3e1800-9d84-11eb-92bc-cbb2fa4c491d.png)
